c/src/sbp.c

```diff
@@ -423,6 +473,12 @@ s8 sbp_process_payload(sbp_state_t *s, u16 sender_id, u16 msg_type, u8 msg_len,
       ret = SBP_OK_CALLBACK_EXECUTED;
     }
   }
+  if (0 != s->allmsg_cb) {
```

Comment:
use `NULL`

Reply:
as discussed, separate PR to be up today